tests/Agent/IntegrationTests/IntegrationTests/CodeLevelMetrics/FrameworkCustomInstrumentationCodeAttributeTests.cs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Does this file need to wait for a log line too?

Reply:
I originally updated this one as well but turns out the test scaffolding can't handle the additional time for this particular test application and starts failing unable to get a wait handle for shutdown. Decided not to dig too deep into what may need to change fundamentally there since it didn't seem to currently be a hard need, just a preventative for a potential issue that could occur. Hopefully doesn't come up but can if does can dig more.

Reply:
Bummer, I was just asking for consistency

Reply:
Yeah, I was surprised :(